docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -91,71 +111,53 @@ The time required for the model to train is proportionate to the amount of data.
 - Best accuracy displays the accuracy of the best performing model found by Model Builder so far. Higher accuracy means the model predicted more correctly on test data.
 - Best algorithm displays the name of the best performing algorithm performed found by Model Builder so far.
 - Last algorithm displays the name of the algorithm most recently used by Model Builder to train the model.
 
-Once training is complete, navigate to the evaluate step.
+Once training is complete, click the **Next step** to navigate to the evaluate step.
 
 ## Evaluate the model
 
-The result of the training step will be one model which had the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the *Best Model* entry along with metrics in *Best Model Quality (RSquared)*. Additionally, a summary table containing top five models and their metrics.
+The result of the training step will be one model which had the best performance. In the evaluate step of the Model Builder tool, in the **Best model** section, will contain the algorithm used by the best performing model in the *Model* entry along with metrics for that model in *RSquared*.
 
-If you're not satisfied with your accuracy metrics, some easy ways to try and improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, navigate to the code step.
+Additionally, in the **Output** window of Visual Studio, there will be a summary table containing top five models and their metrics.
 
-## Add the code to make predictions
+This section will also allow you to test your model by performing a single prediction. It will offer text boxes to fill in values and you can click the **Predict** button to get a prediction from the best model. By default this will be filled in by a random row in your dataset.
 
-Two projects will be created as a result of the training process.
+If you're not satisfied with your accuracy metrics, some easy ways to try and improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, click **Next step** to navigate to the consume step.
 
-- TaxiFarePredictionML.ConsoleApp: A .NET Core Console application that contains the model training and sample consumption code.
-- TaxiFarePredictionML.Model: A .NET Standard class library containing the data models that define the schema of input and output model data, the saved version of the best performing model during training and a helper class called `ConsumeModel` to make predictions.
+## Consume the model
 
-1. In the code step of the Model Builder tool, select **Add Projects** to add the auto-generated projects to the solution.
-1. Open the *Program.cs* file in the *TaxiFarePrediction* project.
-1. Add the following using statement to reference the *TaxiFarePredictionML.Model* project:
+This step will have project templates that you can use to consume the model.
```

Comment:
By this point they'll have the model and everything they need from MB.

Let's make it clear somewhere in here that this step (consume the model) is optional and that it's an EITHER / OR situation, they don't have to do both.